agreementbot/governance.go

```diff
@@ -251,7 +251,7 @@ func (w *AgreementBotWorker) GovernAgreements() int {
 											continue
 										}
 
-										details, err := w.secretProvider.GetSecretDetails(ag.Org, secretUser, secretName)
+										details, err := w.secretProvider.GetSecretDetails(w.EC.Id, w.EC.Token, ag.Org, secretUser, secretName)
```

Comment:
I think the more correct way is this:
GetSecretDetails(w.GetExchangeId(), w.GetExchangeToken(), ...)